<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>debugDialog</class>
 <widget class="QDialog" name="debugDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>449</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Debug插件调试</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>250</height>
    </rect>
   </property>
   <property name="title">
    <string>执行插件</string>
   </property>
   <widget class="QLineEdit" name="exec">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>40</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>60</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>执行指令：</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="threadBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>220</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>使用Thread执行（无返回值）</string>
    </property>
   </widget>
   <widget class="QPushButton" name="execBtn">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>30</y>
      <width>75</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>执行</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>330</width>
      <height>100</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>调试插件时不会初始化一些变量，您可以在这里初始化一些变量</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>附加语句</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>381</width>
     <height>140</height>
    </rect>
   </property>
   <property name="title">
    <string>返回信息</string>
   </property>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>360</width>
      <height>90</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
